hardware.go

```diff
@@ -8,6 +8,15 @@ import (
 	"strings"
 )
 
+// alert can be a buzzer or a light, or something else ...
+func activateAlert() error {
+	_, err = exec.Command("python", "hardware/alert.py").Output()
```

Comment:
Nofication does make more sense, thanks. Not a bad idea for config, although that can be part of a seperate feature/pr